services/metadata/frontend/Menu.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import BurgerButton from "./BurgerButton.svelte";
+
+  let visible = "hidden";
```

Comment:
I would use boolean `isVisible` here.

Reply:
that would have been my first idea, but how would I use it then? (see line 17: https://github.com/dasch-swiss/dasch-service-platform/blob/07918dc5da3e9a106daca7dc56234494d7fc2a40/services/metadata/frontend/Menu.svelte#L17)

Reply:
something like: 
```
<div class="menu" style="isVisible ? 'visible' : 'hidden'">
```

Reply:
something like this:
```
<div class="menu" style="visibility: {toggleVisible() ? 'visible' : 'invisible'}">
```

Besides that, I would avoid adding inline styles. This was always not the best practice. Instead of extra class would be added, or actually 2 classes, which can be swapped. Now it requires Adding those classes here, but should have some base classes in `global.css` file defined either an CSS framework with in-build ones.